packages/api-sdk/tests/resources/project/index.unit.ts

```diff
@@ -188,6 +188,18 @@ describe('ProjectResource', () => {
     expect(data).to.eql(RESPONSE_DATA);
   });
 
+  it('.getVersionsV2', async () => {
+    const { fetch, resource } = createClient();
+
+    fetch.get.resolves({ data: RESPONSE_DATA });
+
+    const data = await resource.getVersionsV2<{ name: string; variables: string[] }>('1');
```

Comment:
missing tests which specify a `limit` and `offset` value. this is why you didn't notice the typings issue from above